CMakeLists.txt

```diff
@@ -76,12 +82,41 @@ if (NOT CMAKE_BUILD_TYPE)
 endif ()
 
 
-################################################################################
+###############################################################################
 # Install prefix settings
-################################################################################
-SET (CMAKE_INSTALL_PREFIX "../install" CACHE STRING
-    "Install path prefix, prepended onto install directories." FORCE)
-MARK_AS_ADVANCED (CMAKE_INSTALL_PREFIX)
+###############################################################################
+set(CMAKE_INSTALL_PREFIX "/usr/local" CACHE PATH
+    "Install path prefix, prepended onto install directories.")
+
+if(NOT IS_ABSOLUTE ${CMAKE_INSTALL_PREFIX})
```

Comment:
I'm not sure I understand why this is important, can you provide your reasoning?

Reply:
This is just to comply with standard behaviour: things by default are installed in `/usr/local`. So to install in the `../install` you need actually to provide the path by hand. 

It's just as the other cmake packages work, not strictly mandatory.

Reply:
OK, was just wondering.